.github/workflows/build.yaml

```diff
@@ -80,6 +80,7 @@ jobs:
         shell: pwsh
         run: |
           cd out/build/${{ matrix.preset }}
+          Import-Module Pester -Force -MinimumVersion '5.6.1' -MaximumVersion '5.99'
```

Comment:
Can you explain what this module is doing?

Reply:
Import Pester for #1512, or I should move the change to that PR?

Reply:
Moved to PR1512